test/games/samus_returns/gui/dialog/test_msr_cosmetic_patches_dialog.py

```diff
@@ -46,6 +46,28 @@ def test_change_music_option(
     assert dialog.cosmetic_patches == MSRCosmeticPatches(music=music_end_value)
 
 
+@pytest.mark.parametrize(
+    ("slider_name", "label_name", "field_name"),
+    [
+        ("music_slider", "music_label", "music_volume"),
+        ("ambience_slider", "ambience_label", "ambience_volume"),
+    ],
+)
+def test_certain_slider(skip_qtbot: pytestqt.qtbot.QtBot, slider_name: str, label_name: str, field_name: str) -> None:
+    cosmetic_patches = MSRCosmeticPatches(**{field_name: 0})  # type: ignore[arg-type]
+
+    dialog = MSRCosmeticPatchesDialog(None, cosmetic_patches)
+    label = getattr(dialog, label_name)
+    skip_qtbot.addWidget(dialog)
+    assert label.text() == "  0%"
+
+    slider = getattr(dialog, slider_name)
+    slider.setValue(80)
+
+    assert dialog.cosmetic_patches == MSRCosmeticPatches(**{field_name: 80})  # type: ignore[arg-type]
+    assert label.text() == " 80%"
```

Comment:
assert that the underlying cosmetic field changes too please

Reply:
I don't know what you mean

Reply:
You only assert that the text field of the slider changes
but we want to make sure that when the slider is changed, that the underlying value (e.g. dialog.cosmetic_patches.ambience_volume)  also changes.

Reply:
dont assert the slider value, assert the underlying field

Reply:
nvm, didnt see that you were already doing that, whoops.